app/src/main/AndroidManifest.xml

```diff
@@ -17,6 +17,7 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
+        android:usesCleartextTraffic="true"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/punchdrunker/hocho/blob/f35064b483037b3ad6a9f92775c86a5acd6b4113/app/src/main/AndroidManifest.xml#L20"></span>Attribute <code>usesCleartextTraffic</code> is only used in API level 23 and higher (current min is 21)</td>
    </tr>
  </tbody>
</table>

Reply:
:expressionless: